Team8923_PowerPlay/src/main/java/org/firstinspires/ftc/team8923_PowerPlay/BlueLeftDetectPark.java

```diff
@@ -0,0 +1,4 @@
+package org.firstinspires.ftc.team8923_PowerPlay;
+
+public class BlueLeftDetectPark {
```

Comment:
Are Left and Right going to be clear to you in the game? Maybe you can use different terminology based on the field.

Reply:
Yes, after looking at the field we can determine which side to start the robot on based on the alliance we're on. If we're blue alliance, we would stand on the blue alliance side and be either left or right and the same for the red alliance. There shouldn't be confusion between the left and right once we decide on our alliance. I hope that makes sense.